packages/destination-actions/src/destinations/fullstory/identifyUser/index.ts

```diff
@@ -57,25 +57,16 @@ const action: ActionDefinition<Settings> = {
   },
   perform: (request, { payload, settings }) => {
     const { traits, anonymousId, userId, email, displayName } = payload
-    let newTraits: Record<string, unknown> = {}
 
-    if (traits) {
-      newTraits = Object.entries(traits).reduce(
-        (acc, [key, value]) => ({
-          ...acc,
-          [camelCaseField(key)]: value
-        }),
-        {}
-      )
-    }
+    const normalizedTraits = normalizePropertyNames(traits, { camelCase: true })
```

Comment:
why do we camel case for identifyUser but not trackEvent?

Reply:
We decided to camel case properties for the `identifyUser` action to match behavior in the browser destination `identifyUser` action even though longer term we'd like to stop unnecessarily changing property names. In contrast, neither the browser destination `trackEvent` action nor the cloud mode destination `trackEvent` action camel case properties.

More context: https://fullstory.slack.com/archives/C02686RAGV6/p1656599360296989